### PR TITLE
3.5 solo

### PR DESCRIPTION
**Pull request by KwakDesign**
The site is responsive to DT (1200px) and mobile (375px).
@omgmakeme
@ltw

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
-    <a href="about.html">About</a>
-  </nav>
-  <h2>My biography:</h2>
-  <p>Born and raised in NJ/NYC area, I began working in the digital media industry 6 years ago. I started as a junior web developer but my career path led me towards user experience design. Today I am working hard to become a software engineer and fulfill my goal to become a designer/engineer hybrid.</p>
+  <div class="main-container">
+    <div class="head-section">
+      <h1><a href="index.html">Abraham Kwak</a></h1>
+      <h3>Product Designer and Full Stack Developer</h3>
+      <nav>
+        <ul id="nav">
```
Comment (ltw) [inline about.html:14]: I love the `ul` inside the `nav` to bring order to the elements inside, but when you're naming your ids it's useful to think "am I going to be able to tell the difference between `nav` and `#nav` at a glance, or is my CSS gonna get _confusing_?"

File: style.css [hunk 2]
```diff
@@ -0,0 +1,398 @@
+* {
```
Comment (ltw) [inline style.css:1]: Nice use of the reset CSS. My favorite reset sheet is Eric Meyer's [Reset CSS 2.0](http://cssreset.com/scripts/eric-meyer-reset-css/) which is worth digging into.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+  -webkit-box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  -moz-box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  width: 100%;
+  min-height:100%;
+  position:relative;
+}
+
+.head-section {
+  padding: 30px;
```
Comment (ltw) [inline style.css:38]: Generally it's good practice to use `em` *OR* `px` (or `rem`) consistently for specifying values in the body. (disclaimer: it's fine to use specific pixel values to describe like `max_width` of the whole page, but then use `em` elsewhere.) 

You're going for consistency so you don't have to translate between measurement systems in your head.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+  margin-top: 50px;
+}
+
+.img-circular{
+  width: 200px;
+  height: 200px;
+  background-image: url('me.jpg');
+  background-size: cover;
+  display: block;
+  border-radius: 200px;
```
Comment (ltw) [inline style.css:148]: I could be wrong, but I'm fairly sure all browsers implement `border-radius` consistently now. Good move to use browser-specific on the box-shadow, but I think this one might be unnecessary these days.

**Review by ltw — COMMENTED** [review]
This is great stuff, @KwakDesign! Couple of things:

1. You may want to look into setting your text editor to add a newline at the end of your files. It's UNIX file convention to have your files end with a newline and it's why Github shows a red arrow thing as the last character on each file in the PR diff.
2. If you ask a specific question in a PR, it allows @omgmakeme and myself to start thinking about your code in a specific context before we answer. If you have no questions but just want to show it off, tag us but we might not find anything to comment on. With a question, you'll most always get an answer.
3. Really good stuff! You're crushing it so far - keep going!

File: style.css (hunk 2 above)
Comment (KwakDesign) [reply style.css:1]: I'll definitely take a look. Thanks for the tip!

**Pull request merged**